lib/chai.js

```diff
@@ -4,14 +4,13 @@
  * MIT Licensed
  */
 
-var used = []
-  , exports = module.exports = {};
+var used = [];
 
 /*!
  * Chai version
  */
 
-exports.version = '3.5.0';
+exports.version = require('../package').version;
```

Comment:
@keithamus Whoops. Revert? Also now that we have more maintainers/reviewers, maybe we should add a minimum time before merging a PR even after it has two approvals.